hw/ip/edn/data/edn.hjson

```diff
@@ -36,6 +30,18 @@
       default: "8",
       local: "true"
     },
+    { name: "BootInsCmd",
```

Comment:
sorry this is kind of unrelated, but this is the command that boot ROM is supposed to supply for initial entropy distribution?

Reply:
ah okay... i checked a bit more... it looks like `boot_req_dis` is 0 by default, but `edn_en` probably is also `0` by default? So i guess that means software just needs to set the latter.

Reply:
yes, the idea here is that boot ROM will do one write to each entropy complex block, and all of the parameters default to a working setting.